Core/Frameworks/Baikal/Controller/Main.php

```diff
@@ -0,0 +1,40 @@
+<?php
+#################################################################
+#  Copyright notice
+#
+#  (c) 2013 Jérôme Schneider <mail@jeromeschneider.fr>
+#  All rights reserved
+#
+#  http://baikal-server.com
```

Comment:
Please change the url to http://sabre.io/baikal

Reply:
fixed in bc84ed7d86b18c84ce83869a27ed6c9876cdbcff
My bad, it was a blind copy of another controller (^_^; )